### PR TITLE
Backend : Adding image URLs into the DB

### LINKED ISSUE #114: Backend : Adding image URLs into the DB
# Description
Added image URLs into the DB

Fixes # (issue)
https://github.com/HackYourFuture-CPH/fp-class19/issues/114

# How to test?
Terminal: npm run db:setup   than  npm run dev

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### PR DESCRIPTION
**Pull request by Impressiv3**
# Description
Added image URLs to development seed file.

Fixes # (issue)
https://github.com/HackYourFuture-CPH/fp-class19/issues/114

# How to test?
Terminal: npm run db:setup

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by m-elshawa — COMMENTED** [review]
Hey, I can see the the picture column on the DB, and you added the pictures to the assets folder, and the branch seems fine, even the frontend working properly. but my question is, aren't these pictures supposed to show on the landing page right now?

**Review by m-elshawa — APPROVED** [review]

File: src/server/seeds/development/01_products_table.js [hunk 1]
```diff
@@ -197,7 +197,7 @@ exports.seed = function (knex) {
           stock_amount: 100,
           is_on_discount: false,
           discount_percent: 0,
-          picture: 'https://via.placeholder.com/500x500',
+          picture: 'src/client/assets/images/image_23.png',
```
Comment (FlorentinaPetica) [inline src/server/seeds/development/01_products_table.js]: image_ 23 and image_26 are missing. But the rest are working and look so nice 👍  And for me the landing page it's showing right with the new pictures.

Comment (Impressiv3) [reply src/server/seeds/development/01_products_table.js]: Thank you Florentina ! 
Changed the missing image's URL.

Comment (FlorentinaPetica) [reply src/server/seeds/development/01_products_table.js]: Hey Balint .... I am sorry but I also said    image_26      , if you change that too than you can merge no problem. 💯

**Pull request merged**